### PR TITLE
Don't list target files that are directly added in a target (.target) while adding

### LINKED ISSUE #159: Don't list target files that are directly added in a target (.target file) while adding
Don't list target files that are directly added in a target (.target file) while adding

### PR DESCRIPTION
**Pull request by vik-chand**
Don't list target files that are directly added in a target (.target) while adding (fixes #159)
Change-Id: If2edf9fbdfc1961b9379936936ad5886e1a05af3
Signed-off-by: Vikas Chandra <Vikas.Chandra@in.ibm.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ pipeline {
 					sh """
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 						-Pbuild-individual-bundles -Pbree-libs -Papi-check \
-						-Dcompare-version-with-baselines.skip=false \
+						-Dcompare-version-with-baselines.skip=true \
```
Comment (akurtakov) [inline Jenkinsfile:22]: This change will make verification build not fail when version bumps are needed thus it sholdn't happen.

**Comment by akurtakov** [comment]
I would suggest keeping the first commit only and rebase it on master to see real test results.

File: Jenkinsfile (hunk 1 above)
Comment (vik-chand) [reply Jenkinsfile]: Yes I was trying out this as I still had the " [ERROR] META-INF/ECLIPSE_.SF: present in baseline only". The intention was to check for other issues and then revert in the next commit  in this PR( as already done)

**Comment by akurtakov** [comment]
Please rebase your branch on master.

**Comment by vik-chand** [comment]
I have rebased on master. When I removed baseline check, mostly unrelated API tool errors were coming.. The current change has only 1 file ( which is required). I will squash and merge this one  ( will track in next I build) and hope I dont end up with "META-INF/ECLIPSE_.RSA: present in baseline only" in the next PR.

**Comment by akurtakov** [comment]
You haven't rebased or at least haven't pushed the rebase . Look at https://github.com/vik-chand/eclipse.pde/tree/issue-159 - "This branch is [5 commits ahead](https://github.com/vik-chand/eclipse.pde/compare/eclipse-pde:master...issue-159), [20 commits behind](https://github.com/vik-chand/eclipse.pde/compare/issue-159...eclipse-pde:master) eclipse-pde:master."

**Comment by vik-chand** [comment]
Revision 2167cbde7f2404f57fa0a60bd50bb89ef95115bb - just the comment change

Revision 81328a4e70306ff4cde1778287329ae9e5f4369d  - with the fix


Both have 349 test failures. In 1st revision there is no change except the comment.
In the 2nd revision, there is the fix and still the same 349 failures.

So the failures are independent.

**Pull request merged**